src/status_im/ui/screens/biometric/views.cljs

```diff
@@ -72,11 +73,9 @@
   (let [keycard-account? @(re-frame/subscribe
                            [:multiaccounts.login/keycard-account?])]
     [biometric-popover
-     {:title-label     :t/biometric-secure-with
-      :ok-button-label :t/biometric-enable-button
-      :on-confirm      :biometric/enable
-
-      :description-label
-      (if keycard-account?
-        (i18n/label :t/biometric-enable-keycard)
-        (i18n/label :t/biometric-enable))}]))
+     {:title-label       :t/biometric-secure-with
```

Comment:
it doesn't look like anything changed here

Reply:
there is still some fixes needed as done here https://github.com/status-im/status-react/pull/9716/files

Reply:
there is no `i18n/label` for :description-label

Reply:
9716 has wrong code, this PR fixed it